ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py

```diff
@@ -422,22 +422,34 @@ def _build_commandlines(self):
             self.num_commands = 0
             return
 
-        # The number of long running commands needed to get data for all
+        def normalize_chunks(chunks, num_commands):
+            if all(len(format_counters(cgroup)) < counters_limit for cgroup in
+                   chunks):
+                return chunks, num_commands
+            else:
+                num_commands += 1
```

Comment:
won't this make the line longer each recursion?  I think it should be num_commands -= 1